workitem/workitem_revision.go

```diff
@@ -6,22 +6,27 @@ import (
 	uuid "github.com/satori/go.uuid"
 )
 
+// RevisionType defines the type of revision for a work item
+type RevisionType int
+
 const (
-	// CREATE_WORKITEM_REVISION_TYPE a work item creation
-	CREATE_WORKITEM_REVISION_TYPE = 1
-	// DELETE_WORKITEM_REVISION_TYPE a work item deletion
-	DELETE_WORKITEM_REVISION_TYPE = 2
-	// UPDATE a work item update
-	UPDATE_WORKITEM_REVISION_TYPE = 4
+	_ RevisionType = iota // ignore first value by assigning to blank identifier
+	// RevisionTypeWorkItemCreate a work item creation
+	RevisionTypeWorkItemCreate // 1
+	// RevisionTypeWorkItemDelete a work item deletion
+	RevisionTypeWorkItemDelete // 2
+	_                          // ignore 3rd value
```

Comment:
With 1,2,4 as flags, you can't mix changes. Eclipse JDT uses these values for the type of changes, for example.